### DISCUSSION TITLE
Add support for mark above/below as read

### DISCUSSION THREAD

**Discussion by onlydole** [community/community#5946] [Mobile]
I'd like to request the ability similar to the Reeder app on iOS where you can long press on a notification and mark all above or below in the list as read. Right now, we have to swipe each issue which is time consuming on mobile. 

![image](https://user-images.githubusercontent.com/117500/135795951-79bf44bd-21dd-4ef1-858f-cc3ada7ef19f.png)

**Comment by brianlovin** [comment]
Thanks for the note @onlydole — I agree. I've added your +1 to our internal tracking issue about how we can make triaging the inbox faster.

**Reply by onlydole** [reply]
Thank you so much, @brianlovin! That sounds fantastic!
